src/main/java/com/minecolonies/coremod/util/EntityUtils.java

```diff
@@ -290,4 +290,28 @@ public static boolean solidOrLiquid(@NotNull final World world, @NotNull final B
         return material.isSolid()
                  || material.isLiquid();
     }
+
+    /**
+     * Get a safe spawnpoint near a location.
+     *
+     * @param nearPoint the point to search near.
+     * @return The spawn position.
+     */
+    @Nullable
+    public static BlockPos getSpawnPoint(World world, BlockPos nearPoint)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document the parameter(s): world [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)